release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -315,6 +315,96 @@ With this release, the `MaxUnavailableStatefulSet` featureset configuration para
+** Added the `PrometheusSDRefreshFailure` alert to monitor failed Prometheus service discoveries.
+
+* *Changed*
+** Modified the `KubeAggregatedAPIDown` alert and the `KubeAggregatedAPIErrors` alert to evaluate only metrics from the `apiserver` job.
+** Modified the `KubeCPUOvercommit` alert to evaluate only metrics from the `kube-state-metrics` job.
+** Modified the `NodeHighNumberConntrackEntriesUsed`, `NodeNetworkReceiveErrs` and `NodeNetworkTransmitErrs` alerts to evaluate only metrics from the `node-exporter` job.
+
+* *Removed*
+** Removed the `MultipleContainersOOMKilled` alert for not being actionable. Nodes under memory pressure are covered by other alerts.
+
+[id="ocp-4-14-monitoring-new-option-to-create-alerts-based-on-core-platform-metrics"]
+==== New option to create alerts based on core platform metrics
+
+With this release, administrators can create new alerting rules based on core platform metrics.
+You can now modify settings for existing platform alerting rules by adjusting thresholds and by changing labels.
+You can also define and add new custom alerting rules by constructing a query expression based on core platform metrics in the `openshift-monitoring` namespace.
+This feature was included as a technology preview in the {product-title} 4.12 release and is now generally available.
+For more information, see xref:../monitoring/managing-alerts.adoc#managing-core-platform-alerting-rules_managing-alerts[Managing alerting rules for core platform monitoring].
+
+[id="ocp-4-14-monitoring-new-option-to-specify-resource-limits-for-all-monitoring-components"]
+==== New option to specify resource limits for all monitoring components
+
+With this release, you can now specify resource requests and limits for all monitoring components, including the following:
+
+* Alertmanager
+* kube-state-metrics
+* monitoring-plugin
+* node-exporter
+* openshift-state-metrics
+* Prometheus
+* Prometheus Adapter
+* Prometheus Operator and its admission webhook service
+* Telemeter Client
+* Thanos Querier
+* Thanos Ruler
+
+In previous versions of {product-title}, you could only set options for Prometheus, Alertmanager, Thanos Querier, and Thanos Ruler.
+
+[id="ocp-4-14-monitoring-new-options-to-configure-node-exporter-collectors"]
+==== New options to configure node-exporter collectors
+
+With this release, you can customize Cluster Monitoring Operator (CMO) config map settings for additional node-exporter collectors.
+The following node-exporter collectors are now optional, and you can enable or disable each one individually in the config map settings:
+
+* `ksmd` collector
+* `mountstats` collector
+* `processes` collector
+* `systemd` collector
+
+In addition, you can now exclude network devices from the relevant collector configuration for the `netdev` and `netclass` collectors.
+You can also now use the `gomaxprocs` option to set the maximum number of processes that can run node-exporter.
```

Comment:
```suggestion
You can also now use the `maxProcs` option to set the maximum number of processes that can run node-exporter.
```
given the setting, I think `maxProcs` is better, let @raptorsun decide
```
 config.yaml: |
    nodeExporter: 
      maxProcs: 1
```

Reply:
I would use `maxProcs` here as @juzhao suggested.

Reply:
changed